TAF_EasyRest/TestFramework/PageComponents/NavigationMenuPageComponent.cs

```diff
@@ -10,6 +10,7 @@ public NavigationMenuPageComponent(IWebDriver driver) : base(driver)
 
         protected static TimeSpan timeout = TimeSpan.FromSeconds(3);
         protected static WebDriverWait wait;
+        UserMenuDropDownListPageComponent userMenuDropDownListPageComponent;
```

Comment:
This component is unavailable. If you want to create it in the future - make a cap like a mock class UserMenuDropDownListPageComponent, so in this class everything will be ok.

Reply:
...and use an access modifier also.